src/main/java/GeneralService/OnlineServicesCommunicationService.java

```diff
@@ -0,0 +1,51 @@
+package GeneralService;
+
+import onlineservices.services.WindowControl.WindowControlService;
+import org.eclipse.paho.client.mqttv3.*;
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OnlineServicesCommunicationService {
+
+    private static final String MQTT_BROKER_URL = "tcp://broker.emqx.io:1883";
+    private static final String MQTT_CLIENT_ID = "OnlineServicesCommunicationService";
+    private static final String MQTT_WINDOW_SERVICE_TOPIC = "WindowControlService";
+    private MqttClient windowControlServiceClient;
```

Comment:
this should be called mqttClient, we will be using 1 client to communicate with all the services, if needed